docs/how-to/heterogeneous_matches.rst

```diff
@@ -0,0 +1,65 @@
+.. _heterogeneous-matches:
+
+Heterogeneous Matches
+=====================
```

Comment:
```suggestion
Use custom matches
================
```

Then if you do want to use the mass based example from the paper you worked on @zewaywong you should include that as a reference (in `reference/bibliography.rst` which will appear here: https://axelrod.readthedocs.io/en/latest/reference/bibliography.html).

Perhaps first with something like:

```
Similarly to [<your paper>] we are going to assign a mass to each individual for a Moran process.
This requires us to build a specific match class:

    <code for MassBasedMatch>

In [<your paper>] a non standard Moran process is used where the mass of individuals is not reproduced so we will use inheritance to create a new Moran process that keeps the mass of the individuals constant.

    <code where you inherit a new Moran process>
```

You could also have both examples with essentially 3 different code snippets:

1. Creating a new match class
2. Running the base Moran process where the mass is not constant.
3. Running a modified Moran process where the mass **is** constant and you see how that changes.